core/services/keystore/keys/ocr2key/evm_key_bundle.go

```diff
@@ -1,123 +1,123 @@
 package ocr2key
 
-import (
-	cryptorand "crypto/rand"
-	"crypto/sha256"
-	"encoding/hex"
-	"encoding/json"
-	"io"
-
-	"github.com/pkg/errors"
-	"github.com/smartcontractkit/chainlink/core/services/keystore/chaintype"
-)
-
-type (
-	// evmKeyBundle represents the bundle of keys needed for OCR
-	evmKeyBundle struct {
-		keyBundleBase
-		evmKeyring
-	}
-
-	evmKeyBundleRawData struct {
-		ChainType       chaintype.ChainType
-		OffchainKeyring []byte
-		EVMKeyring      []byte
-	}
-)
-
-var _ KeyBundle = &evmKeyBundle{}
-
-func newEVMKeyBundle() (*evmKeyBundle, error) {
-	return newEVMKeyBundleFrom(cryptorand.Reader, cryptorand.Reader, cryptorand.Reader)
-}
-
-func mustNewEVMKeyBundleInsecure(reader io.Reader) *evmKeyBundle {
-	key, err := newEVMKeyBundleFrom(reader, reader, reader)
-	if err != nil {
-		panic(errors.Wrap(err, "failed to generate new OCR2-EVM Key"))
-	}
-	return key
-}
-
-func newEVMKeyBundleFrom(onchainSigningKeyMaterial, onchainEncryptionKeyMaterial, offchainKeyMaterial io.Reader) (*evmKeyBundle, error) {
-	offchainKeyring, err := newOffchainKeyring(onchainSigningKeyMaterial, onchainEncryptionKeyMaterial)
-	if err != nil {
-		return nil, err
-	}
-	evmKeyRing, err := newEVMKeyring(onchainSigningKeyMaterial)
```

Comment:
noticed that the `newOffchainKeyring` and `newEVMKeyring` both use `onchainSigningKeyMaterial`. was that intentional? seems like the offchain keyring should be using `offchainKeyMaterial`

Reply:
@RyanRHall is there a chance you could help us get to the bottom of this?

Reply:
also I think if we were to change this, it shouldn't affect existing keys since it's used in generating new keys